tests/modules/programs/i3status-rust/with-old-state-version.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
```suggestion
```

Not being used.

Reply:
Removed it from all tests since they're not being used too.